modules/hrm/views/js-templates/announcement-more-sent-to.php

```diff
@@ -0,0 +1 @@
+{{{data.all_send_to}}}
```

Comment:
Why are you using a new template for just this one line? Check if it can be done without a new template file.